cores/esp8266/timehelper.cpp

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
This might better be put into the existing `time.cpp` file instead of making a new one.

Also, a new file would need the GPL headers/etc.

Reply:
> This might better be put into the existing time.cpp file instead of making a new one.

`time.cpp` is not compiled in host mode. This new file would.

Reply:
The function is now in `time.cpp`, the other new file is removed.